components/mpas-ocean/bld/namelist_files/namelist_defaults_mpaso.xml

```diff
@@ -755,6 +756,34 @@
 <config_AM_timeSeriesStatsClimatology_reset_intervals>'1000-00-00_00:00:00;1000-00-00_00:00:00;1000-00-00_00:00:00;1000-00-00_00:00:00'</config_AM_timeSeriesStatsClimatology_reset_intervals>
 <config_AM_timeSeriesStatsClimatology_backward_output_offset>'00-03-00_00:00:00'</config_AM_timeSeriesStatsClimatology_backward_output_offset>
 
+<!-- AM_timeSeriesStatsMonthlyMax -->
+<config_AM_timeSeriesStatsMonthlyMax_enable>.false.</config_AM_timeSeriesStatsMonthlyMax_enable>
+<config_AM_timeSeriesStatsMonthlyMax_compute_on_startup>.false.</config_AM_timeSeriesStatsMonthlyMax_compute_on_startup>
+<config_AM_timeSeriesStatsMonthlyMax_write_on_startup>.false.</config_AM_timeSeriesStatsMonthlyMax_write_on_startup>
+<config_AM_timeSeriesStatsMonthlyMax_compute_interval>'00-00-00_01:00:00'</config_AM_timeSeriesStatsMonthlyMax_compute_interval>
+<config_AM_timeSeriesStatsMonthlyMax_output_stream>'timeSeriesStatsMonthlyMaxOutput'</config_AM_timeSeriesStatsMonthlyMax_output_stream>
+<config_AM_timeSeriesStatsMonthlyMax_restart_stream>'timeSeriesStatsMonthlyMaxRestart'</config_AM_timeSeriesStatsMonthlyMax_restart_stream>
+<config_AM_timeSeriesStatsMonthlyMax_operation>'max'</config_AM_timeSeriesStatsMonthlyMax_operation>
+<config_AM_timeSeriesStatsMonthlyMax_reference_times>'initial_time'</config_AM_timeSeriesStatsMonthlyMax_reference_times>
+<config_AM_timeSeriesStatsMonthlyMax_duration_intervals>'repeat_interval'</config_AM_timeSeriesStatsMonthlyMax_duration_intervals>
+<config_AM_timeSeriesStatsMonthlyMax_repeat_intervals>'reset_interval'</config_AM_timeSeriesStatsMonthlyMax_repeat_intervals>
+<config_AM_timeSeriesStatsMonthlyMax_reset_intervals>'00-01-00_00:00:00'</config_AM_timeSeriesStatsMonthlyMax_reset_intervals>
+<config_AM_timeSeriesStatsMonthlyMax_backward_output_offset>'00-01-00_00:00:00'</config_AM_timeSeriesStatsMonthlyMax_backward_output_offset>
+
+<!-- AM_timeSeriesStatsMonthlyMin -->
+<config_AM_timeSeriesStatsMonthlyMin_enable>.false.</config_AM_timeSeriesStatsMonthlyMin_enable>
+<config_AM_timeSeriesStatsMonthlyMin_compute_on_startup>.false.</config_AM_timeSeriesStatsMonthlyMin_compute_on_startup>
+<config_AM_timeSeriesStatsMonthlyMin_write_on_startup>.false.</config_AM_timeSeriesStatsMonthlyMin_write_on_startup>
+<config_AM_timeSeriesStatsMonthlyMin_compute_interval>'00-00-00_01:00:00'</config_AM_timeSeriesStatsMonthlyMin_compute_interval>
+<config_AM_timeSeriesStatsMonthlyMin_output_stream>'timeSeriesStatsMonthlyMinOutput'</config_AM_timeSeriesStatsMonthlyMin_output_stream>
+<config_AM_timeSeriesStatsMonthlyMin_restart_stream>'timeSeriesStatsMonthlyMinRestart'</config_AM_timeSeriesStatsMonthlyMin_restart_stream>
```

Comment:
@jonbob it looks like the reason I get output of restart is here.  The default should be 'none' to disable writing the restart.